internal/alpha/printers/table/printer.go

```diff
@@ -161,23 +162,30 @@ func getProgress(resInfo *resourceInfo) (string, string, error) {
 	var details string
 	switch resInfo.ResourceAction {
 	case event.ApplyAction:
-		switch resInfo.ApplyStatus {
-		case event.ApplyPending:
-			text = "PendingApply"
-		case event.ApplySuccessful:
-			text = "Applied"
-			printStatus = true
-		case event.ApplySkipped:
-			text = "SkippedApply"
-		case event.ApplyFailed:
-			text = "ApplyFailed"
+		{
```

Comment:
Why the extra set of brackets around this switch statement and not the next one?

Reply:
Good point - I love my unnecessary brackets!  Removed.